src/common/low_precision_transformations/tests/convolution_with_incorrect_weights.cpp

```diff
@@ -62,9 +62,8 @@ class ConvolutionWithIncorrectWeightsTransformation
         SimpleLowPrecisionTransformer transform;
         transform.add<ngraph::pass::low_precision::ConvolutionTransformation, ngraph::opset1::Convolution>(
             testValues.params);
-        transform
-            .add<ngraph::pass::low_precision::FakeQuantizeDecompositionTransformation, ngraph::opset1::FakeQuantize>(
-                testValues.params);
+        transform.add<ngraph::pass::low_precision::FakeQuantizeDecompositionTransformation, ov::op::v0::FakeQuantize>(
```

Comment:
Just question
is here any tickets to move `ngraph::pass::low_precision::FakeQuantizeDecompositionTransformation` -> `ov::pass::low_precision::FakeQuantizeDecompositionTransformation`? If no, could you please create?

Reply:
There is a main jira [CVS-111358](https://jira.devtools.intel.com/browse/CVS-111358) for whole test refactor activity.
I will create a sub task about refactoring certain tests